libtransmission/peer-mse.cc

```diff
@@ -58,8 +58,8 @@ auto export_bits(UIntWide i)
 }
 
 // NOLINTBEGIN(readability-identifier-naming)
-auto WIDE_INTEGER_CONSTEXPR const generator = wi::key_t{ "2" };
-auto WIDE_INTEGER_CONSTEXPR const prime = wi::key_t{
+auto const generator = wi::key_t{ "2" };
+auto const prime = wi::key_t{
```

Comment:
Why was this change needed?

Reply:
`constexpr`-ness inside this library (wide_integer) fails to compile with C++20.
I think I could try to check upstream for this issue, maybe there's a fix already implemented and a simple dependency update will fix this.

Reply:
Actually, works for me now with AppleClang 15.0.0 and LLVM Clang 18.0.6 on macOS.
I think I had problems with GCC 13.x on Linux.